symfony/plugins/orangehrmPerformancePlugin/Service/PerformanceReviewService.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+namespace OrangeHRM\Performance\Service;
+use Doctrine_Collection;
+use OrangeHRM\Performance\Dao\KpiDao;
+use OrangeHRM\Performance\Dao\PerformanceReviewDao;
+use PerformanceReview;
+use sfDoctrineRecord;
+use type;
+
+/**
+ * Description of PerformanceReviewService
+ *
+ * @author nadeera
+ */
+class PerformanceReviewService
+{
+
+    public $dao;
+
+    /**
+     *
+     * @return PerformanceReviewDao
+     */
+    public function getDao()
+    {
+        if ($this->dao != null) {
+            return $this->dao;
+        } else {
+            return new PerformanceReviewDao();
+        }
+    }
+
+    /**
+     *
+     * @param KpiDao $dao
+     */
+    public function setDao($dao)
+    {
+        $this->dao = $dao;
+    }
```

Comment:
remove

Reply:
Reverted this file since it is not necessary for the current changes